packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -192,22 +190,20 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     /* @conditional-compile-remove(data-loss-prevention) */ message.messageType !== 'blocked';
   const [messageReadBy, setMessageReadBy] = useState<{ id: string; displayName: string }[]>([]);
 
-  const actionMenuProps = wasInteractionByTouch
-    ? undefined
-    : chatMessageActionMenuProps({
-        ariaLabel: strings.actionMenuMoreOptions ?? '',
-        enabled: chatActionsEnabled,
-        menuButtonRef: messageActionButtonRef,
-        // Force show the action button while the flyout is open (otherwise this will dismiss when the pointer is hovered over the flyout)
-        forceShow: chatMessageActionFlyoutTarget === messageActionButtonRef,
-        onActionButtonClick: () => {
-          if (message.messageType === 'chat') {
-            props.onActionButtonClick(message, setMessageReadBy);
-            setChatMessageActionFlyoutTarget(messageActionButtonRef);
-          }
-        },
-        theme
-      });
+  const actionMenuProps = chatMessageActionMenuProps({
```

Comment:
iOS: The second tap on message (not menu button) opens the edit menu. Is it expected?

Reply:
To be addressed in the separate PR

Reply:
Created separate issue for tracking this.